app/models/webhook_endpoint.rb

```diff
@@ -2,5 +2,8 @@
 
 # Records a webhook url to send notifications to
 class WebhookEndpoint < ApplicationRecord
+  WEBHOOK_TYPES = %w(order_cycle_opened payment_status_changed).freeze
+
   validates :url, presence: true
+  validates :webhook_type, presence: true, inclusion: { in: WEBHOOK_TYPES }
```

Comment:
The column name "webhook_type" has duplication with the table name. 
But `type` is reserved by ActiveRecord for polymorphic tables. Hmm should this table be polymorphic (that is, should we have sub-classes like `Webhook::OrderCycleOpened`)?

Reply:
We could, but at this stage there is no really difference between the two type of `Webhook` so it seems unnecessary.